Marlin/src/HAL/STM32F1/timers.h

```diff
@@ -80,7 +80,7 @@ typedef uint16_t hal_timer_t;
   //#define TEMP_TIMER_NUM      4  // 2->4, Timer 2 for Stepper Current PWM
 #endif
 
-#if MB(BTT_SKR_MINI_E3_V1_0, BTT_SKR_E3_DIP, BTT_SKR_MINI_E3_V1_2, MKS_ROBIN_LITE)
+#if MB(BTT_SKR_MINI_E3_V1_0, BTT_SKR_MINI_E3_V1_2, BTT_SKR_MINI_E3_V2_0, BTT_SKR_E3_DIP, MKS_ROBIN_LITE)
```

Comment:
It looks like both fan pins on this board use timer 8. That means this PR should introduce a problem, rather than solve one.

More work is probably needed to identify why this seems to solve the submitter’s problem, because this looks incorrect to me.

Reply:
???
Using the same timer means using the same PWM-frequency - not using the the same PWM-ratio what's defined per pin.
Why shouldn't the fans share the same PWM-frequency? Do we have a way to configure different frequencies for different fans?

Reply:
This is attempting to resolve a conflict between fans and Servo output, not between the two fans. Setting fan values when they share a timer with servo output corrupts the servo output frequency.